components/script/dom/range.rs

```diff
@@ -343,7 +343,7 @@ impl RangeMethods for Range {
         } else {
             // Step 3.
             self.set_start(node, offset);
-            if !(self.start() <= self.end()) {
+            if self.start().partial_cmp(self.end()) == Some(Ordering::Greater) {
```

Comment:
Why not change this to `if self.start() > self.end()` ?

Reply:
Thank you @mrobinson for looking at this.
I had used ```partial_cmp``` in case self.start() or self.end() is a none integer number but looking at ```offset: u32``` , I can say it's an integer.
So yes sir, ```if self.start() > self.end() ``` is the best pick here.

Thanks again.